h2o-py/docs/modeling.rst

```diff
@@ -67,12 +67,6 @@ Supervised
     :show-inheritance:
     :members:
 
-:mod:`H2OXGBoostEstimator`
```

Comment:
why is this removed?

Reply:
> smalldata is not kept in git, we can add these to our s3 storage

The dataset was already added and the Jenkins job is running, thanks to @Pscheidl .

Reply:
Maybe some accident? @valenad1, can you resolve this?

Reply:
It is weird, when I build the master branch from 21.8 then the H2OXGBoostEstimator is again removed from the file and from h2o-py/h2o/estimators/__init__.py. Because the files are generated I told my self that maybe someone forgot to commit them. If you say that H2OXGBoostEstimator should be in the file then it is probably something wrong with my dev environment.

Reply:
It's a general problem on windows, will be fixed in https://github.com/h2oai/h2o-3/pull/4907